acnportal/acnsim/network/charging_network.py

```diff
@@ -22,16 +31,16 @@ def current_charging_rates(self):
             Dict[str, number]: Dictionary mapping station_id to the current actual charging rate of the EV attached to
                 that EVSE.
         """
-        current_rates = {}
-        for station_id, evse in self._EVSEs.items():
+        current_rates = np.zeros(len(self._EVSEs))
```

Comment:
current_rates is no longer a dictionary. Either we should make it one or update the docs. Perhaps it should be a Series since that is how we store charging_rates, etc.

Reply:
Kept `current_rates` as a numpy array and updated the docs, since `current_charging_rates` is used only once internally in `simulator`, and there it's assumed to be a numpy array.